webapp/src/assets.ts

```diff
@@ -70,6 +70,13 @@ export function getAssets(gallery = false, firstType = pxt.AssetType.Image, temp
         }
     }
 
+    // sort so that non-builtin assets appear at the top of the gallery
+    images.sort(comparePackage);
+    tiles.sort(comparePackage);
+    tilemaps.sort(comparePackage);
```

Comment:
Do we support extensions for tilemap packs and/or do we have builtin tilemaps? Wondering if it's necessary for me to sort here. And if so, then I think I'd need to add the package info as well like in L1032 of tilemap.ts.

Reply:
songs and tilemaps don't currently have galleries so they couldn't be pulled in from extensions, but I'd probably support it if it's easy (which it should be when it's just adding a field when loading) as otherwise this sort of thing will be easily forgotten in the future if we do choose to add galleries for them.